docs/source/advanced/children.rst

```diff
@@ -0,0 +1,57 @@
+Children
+========
+
+Child targets are nested targets. Targets can contain nested targets, this is almost always the case if
+the target is an hypervisor. After retrieving a child target, it behaves just like a regular target, all
+the plugin methods are available. The following child targets are supported:
+
+* ESXi
+* Hyper-V 
+* Virtuozzo
+* VMWare
+* WSL
+
+To obtain a list of nested targets use ``list_children``:
```

Comment:
Is it a good idea to directly start with listing API methods? Should you not start with introducing how to interface with it from e.g. `target-query`?